internal/pkg/api/server/templates/http/middleware.go

```diff
@@ -41,6 +41,10 @@ func TraceEndpointsMiddleware() func(endpoint goa.Endpoint) goa.Endpoint {
 				}()
 			}
 
+			// Add dependencies to the context
+			reqDeps := dependencies.NewDepsForPublicRequest(serverDeps, ctx, ctx.Value(middleware.RequestIDKey).(string))
+			ctx = context.WithValue(ctx, dependencies.ForPublicRequestCtxKey, reqDeps)
+
 			// Handle
```

Comment:
Prvy commit ... `dependencies` musime napojit na `endpoint context/span`, nie na `request context/span`.

Predtym:
`request span -> dependencies -> d.RequestCtx() -> operations ....`
![image](https://user-images.githubusercontent.com/19371734/188813412-8f42ba9a-5a24-419a-9ee6-ba2e81b77755.png)




Teraz:
`request span -> endpoint span -> dependencies -> d.RequestCtx() -> operations ...`:
![image](https://user-images.githubusercontent.com/19371734/188813732-8fc0a5b4-37e7-4345-96e2-42353e9693a0.png)

----------

Tie male `spans` vlavo je nacitanie projektu, musime zistit co je to biele miesto, vid dasli commit.